doc/source/installation.rst

```diff
@@ -4,7 +4,7 @@
 Installing |Percona Server| 5.7
 ===============================
 
-This page provides the information on how to you can install |Percona Server|. Following options are available: 
+This page provides the information on how to you can install |Percona Server|. Following options are available:
```

Comment:
`This page provides the information on how to you can install` maybe `This page provides the information on how you can install`

Reply:
fixed.